### ISSUE TITLE
Long dependency queue support

### ISSUE THREAD

**Issue by rafaelverger** [raphamorim/nautilus.js#4]
If I have 3 scripts to load (`jquery`, `jquery-ui`, `internal-ui-script`), I can't set a dependency queue that forces `internal-script-ui` to be loaded at the expected moment;

Right now I have to do something like this:
```js
nautilus('jquery', ['jquery-ui'], () => nautilus('internal-script-ui'));
```

I think that nautilus could assume that its last param (`arguments[arguments.length-1]`) is the `depsOrFn` var, the first is the first script's array and then treats every other param as a ordered dependency queue.

This change makes sense?

**Comment by raphamorim** [comment]
Hey @rafaelverger 

What do you think about?
```js
nautilus('jquery', ['jquery-ui'], ['internal-script-iu'], function(){
   // put your logic here
})
```

Make sense to you? If yes, let me know to start this feature.

**Comment by raphamorim** [comment]
Other thing I was thinking (tehehe)

Is to use nautilus config to set who is dependent. Like

```
nautilus.config({
     'jquery-ui': {
         depends: 'jquery'
     }
})

nautilus('jquery-ui', ['internal-script-ui'])
```

**Comment by rafaelverger** [comment]
Hmm.... the first comment you made is exactly what I was thinking (and tried to explain)... I can also made a PR implementing this feature :)

**Comment by raphamorim** [comment]
Fixed in 08544baf0fa75f580763ca7b4e0116d77ebd6498
